src/analyze/analysis_handler.go

```diff
@@ -9,23 +9,55 @@ import (
 	"time"
 )
 
+var AllResults []common.Result //global so that it can be added onto by parser and seen by dataHandler
+
 // Takes in a Window, parses object into frequency counts, and then calls specified analysis functions
-func AnalyzeBGPMessages(window common.Window) {
+//code to write the frequencies; the outliers; and the minReqs to files
+func AnalyzeBGPMessages(window common.Window) common.Result {
 	bucketMap := window.BucketMap
 
 	// Convert BucketMap to a map of timestamp to length of messages
 	lengthMap := make(map[time.Time]float64)
+	timeStampsFull := []string{}
 
 	for timestamp, messages := range bucketMap {
 		lengthMap[timestamp] = float64(len(messages))
+		timeStampsFull = append(timeStampsFull, timestamp.Format("2006-01-02 15:04:05")) //double check if the formatting is ok
 	}
+	fmt.Println(timeStampsFull)
 
 	// Turn map into sorted array of frequencies by timestamp
 	sortedFrequencies := GetSortedFrequencies(lengthMap)
 
+	//the file names will contain all the timestamps for a given folder that was processed
 	fmt.Printf("Sorted Array of Frequencies: \n%+v\n", sortedFrequencies)
-	fmt.Printf("BLT MAD Outliers: \n%+v\n", blt_mad.BltMad(sortedFrequencies, 10))
+	fmt.Printf("BLT MAD Outliers: \n%+v\n", blt_mad.BltMad(sortedFrequencies, 5))
 	fmt.Printf("ShakeAlert Outliers: \n%+v\n", shake_alert.FindOutliers(sortedFrequencies))
+
+	//put all the results into the Result struct and pass write it out to a json
+	r := common.Result{
+		Frequencies:          sortedFrequencies,
+		MADOutliers:          blt_mad.BltMad(sortedFrequencies, 5),
+		MADTimestamps:        make([]time.Time, 0), //ask about how to actually get these
+		ShakeAlertOutliers:   shake_alert.FindOutliers(sortedFrequencies),
+		ShakeAlertTimestamps: make([]time.Time, 0),
+	}
+	blt_mad.StoreResultIntoJson(r, "static_data/result.json")
+	maxPoints := 100
```

Comment:
make this a constant at the top. That way you dont need to dig through the function to change it.